configure.ac

```diff
@@ -31,6 +31,23 @@ AC_SUBST(DOMJUDGE_VERSION, $PACKAGE_VERSION)
 AC_DEFINE_UNQUOTED(DOMJUDGE_VERSION, "$PACKAGE_VERSION",
 [DOMjudge version number (alias for PACKAGE_VERSION)])
 
+DOMJUDGE_VERSION_MIMOR=m4_esyscmd([grep 'version' README.md | sed -n '1s/^.*version //p' | sed 's/DEV$//' | sed 's/\.[[:digit:]]*$//'])
+DOMJUDGE_IS_DEV=m4_esyscmd([grep 'version' README.md | sed -n '1s/^.*version //p' | sed 's/[[:digit:]]//g;s/\.//g'])
+
+AC_SUBST(DOMJUDGE_VERSION_MIMOR, $DOMJUDGE_VERSION_MIMOR)
```

Comment:
There's a typo `MIMOR`.

But besides that, I think it's better to not put all this logic here, but in the import-script instead: just parse the full `DOMJUDGE_VERSION` string there and determine the URL from it. Then the changes in `configure.ac` and `paths.mk.in` are not needed.

Reply:
How do I do that? It’s a comment in python so i can’t really do that? Or does subst have some magic?

Reply:
Oh, I didn't look at the python part. I don't know whether you can create dynamic doc strings there.

But if all of this is just about getting that doc string to point to the exact URL, then maybe this is not worth it?

Reply:
That is good enough yeah